connect.js

```diff
@@ -184,18 +174,18 @@ defaultLogger.log = (cat, ...args) => {
   console.log(`stripes-connect (${cat})`, ...args);
 };
 
-export const connect = (Component, module, epics, loggerArg, options) => {
+export const connect = (Component, module, epics, loggerArg, addReducer, store, options) => {
```

Comment:
This is a backwards-incompatible change to the `connect` API, which makes it unacceptable. Code using the old API will be passing in a set of options, which the new code will interpret as its addReducer function.

Instead, we need to either add the new parameters at the end of the list (where they will simply be ignored if new client code calls an old version), or add a new `connect2` API.

Reply:
The same also applies in other places in the code.

Reply:
In general, we need to be super-careful about backwards compatibility, especially as more and more people come onto the FOLIO team and we start finding ourselves in the position of needing to continue to support client code written by third parties -- some of which we might not even know about, let alone be in a position to update.

Reply:
Fixed in latest commit, so that the legacy context API is used as a backup